app/views/comments/_comment_proper.html.erb

```diff
@@ -68,6 +68,14 @@
       </div>
 
       <div class="body <%= "low-quality-comment" if decorated_comment.low_quality %>">
+        <% if decorated_comment.display_edited? %>
+          <div class="comment-edited-notice comment-date">
+            Edited
+            <time datetime="<%= decorated_comment.edited_timestamp %>">
+              <%= decorated_comment.edited_at %>
+            </time>
+          </div>
```

Comment:
This should probably resemble as much as possible the wording and date format of how we display this info for articles:

![Screenshot_2019-11-05  Nine Coaches Waiting](https://user-images.githubusercontent.com/146201/68195770-bdcdf400-ffb7-11e9-8bfa-0f59ae2ad02d.png)

So it should say "Updated on [DATE]", where date is "Nov 05, 2019" instead of what it looks like now:

![Screenshot_2019-11-05 Discussion of Nine Coaches Waiting](https://user-images.githubusercontent.com/146201/68195855-ebb33880-ffb7-11e9-9386-f8fe653d363e.png)

Basically by calling `decorated_comment.edited_at.strftime("%b %d, %Y")` instead of `decorated_comment.edited_at` for the visible date